server/sandbox_run_linux.go

```diff
@@ -144,10 +142,11 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 	g.SetProcessArgs(pauseCommand)
 
 	// set DNS options
-	if req.GetConfig().GetDnsConfig() != nil {
-		dnsServers := req.GetConfig().GetDnsConfig().Servers
-		dnsSearches := req.GetConfig().GetDnsConfig().Searches
-		dnsOptions := req.GetConfig().GetDnsConfig().Options
+	var resolvPath string
+	if sbox.Config().GetDnsConfig() != nil {
+		dnsServers := sbox.Config().GetDnsConfig().Servers
+		dnsSearches := sbox.Config().GetDnsConfig().Searches
+		dnsOptions := sbox.Config().GetDnsConfig().Options
 		resolvPath = fmt.Sprintf("%s/resolv.conf", podContainer.RunDir)
```

Comment:
```suggestion
		resolvPath := fmt.Sprintf("%s/resolv.conf", podContainer.RunDir)
```
I think then we can remove line 145.

Reply:
`resolvePath` is use outside this block. Further down in line 327.

Reply:
Ah alright thanks for the clarification